src/EmojiTranslatorContainer.js

```diff
@@ -16,10 +24,21 @@ export default class EmojiTranslatorContainer extends Component {
   handleTranslate(event) {
     this.setState({
       originalContent: event.target.value,
-      translatedContent: event.target.value.toUpperCase()
+      //translatedContent: event.target.value.toUpperCase()
+      translatedContent: this.mapOC(event.target.value)
```

Comment:
instead of storing the original content and translated content on the state, you probably can store only the original content and then translate it before render. That way you only need to store one piece of state.